code/backend/Odin/src/main/kotlin/pt/isel/odin/model/Course.kt

```diff
@@ -0,0 +1,28 @@
+package pt.isel.odin.model
+
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+/**
+ * Represents a course in the system
+ * @property id the course id
+ * @property name the course name
+ * @property descriptions the course description
+ * @property type the course type, is a [CourseType]
+ */
```

Comment:
I'm personally not a big fan of 'useless' comments, this comment add very little value to the codebase as they're self explanatory.
If you wish to keep this you will follow this pattern for all objects.

I advise you to chose wisely.